braindecode/scoring.py

```diff
@@ -12,9 +12,79 @@
 from skorch.utils import to_numpy
 from skorch.dataset import unpack_data
 
+from braindecode.monitors import compute_preds_per_trial_from_trial_n_samples
 from .monitors import compute_preds_per_trial_from_crops
 
 
+def trial_preds_from_supercrop_preds(
+        preds, i_supercrop_in_trials, i_stop_in_trials):
+    """
+    Assigning supercrop predictions to trials  while removing duplicate
+    predictions.
+
+    Parameters
+    ----------
+    preds: list of ndarrays (atleast 2darrays)
+        List of supercrop predictions, in each supercrop prediction
+         time is in axis=1
+    i_supercrop_in_trials: list
```

Comment:
maybe we can switch from `i` to `idx` for all those variables? for me it would be more clear

Reply:
Created an issue for such questions in general https://github.com/braindecode/braindecode/issues/59